### DISCUSSION TITLE
Icon not working on WSL2

### DISCUSSION THREAD

**Discussion by yensubldg** [nvim-tree/nvim-tree.lua#1536] [Q&A] [answered]
Hi!
I add `nvim-tree` and `nvim-web-devicons` to my `nvim`

My nvim display 

![image](https://user-images.githubusercontent.com/63493697/185778507-cc3fd310-4c56-4edd-8bc5-51c11d217c0d.png)

My config in `.vimrc`

![image](https://user-images.githubusercontent.com/63493697/185778519-e9617995-2a92-41ce-8cfd-152ae41506a9.png)
![image](https://user-images.githubusercontent.com/63493697/185778524-71ebc7e7-bb21-4526-bb21-96d35b254912.png)

Can you tell me if I have misconfigured here or is this my use on windows `wsl` environment?

**Comment by yensubldg** [comment, marked as answer]
I config font in setting for Ubuntu in WSL2 and it work

![image](https://user-images.githubusercontent.com/63493697/185780009-85fd66ea-777d-4287-a3ba-4be44539e2a9.png)

![image](https://user-images.githubusercontent.com/63493697/185780012-10ffe23a-0290-4311-b6a0-62328c4fbf3d.png)

**Comment by alex-courtis** [comment]
It looks like you've solved your issue. Moving to discussion as the font setting image is useful.

**Comment by yensubldg** [comment]
So awesome
